package.json

```diff
@@ -3,11 +3,11 @@
 	"version": "0.4.0",
 	"private": true,
 	"scripts": {
-		"build": "npm run clean && env BROWSERSLIST_CONFIG=./package.json rollup -c",
-		"build:light": "npm run clean && env BROWSERSLIST_CONFIG=./package.json rollup -c --light_mode",
-		"dev": "npm run clean && env BROWSERSLIST_CONFIG=./package.json rollup -c -w",
-		"dev:light": "npm run clean && env BROWSERSLIST_CONFIG=./package.json rollup -c -w --light_mode",
-		"start": "sirv dist --single",
+		"build": "npm run clean && env NODE_ENV=production BROWSERSLIST_CONFIG=./package.json rollup -c rollup.config.ts --configPlugin @rollup/plugin-typescript",
+		"build:light": "npm run clean && env NODE_ENV=production BROWSERSLIST_CONFIG=./package.json LIGHT_MODE=1 rollup -c rollup.config.ts --configPlugin @rollup/plugin-typescript",
+		"dev": "npm run clean && env NODE_ENV=development BROWSERSLIST_CONFIG=./package.json rollup -w -c rollup.config.ts --configPlugin @rollup/plugin-typescript",
+		"dev:light": "npm run clean && env NODE_ENV=development BROWSERSLIST_CONFIG=./package.json LIGHT_MODE=1 rollup -w -c rollup.config.ts --configPlugin @rollup/plugin-typescript",
+		"start": "sirv dist --single --host",
```

Comment:
Maybe `sirv dist --single --host 0.0.0.0`?

Reply:
By default the host is 0.0.0.0 [see](https://github.com/lukeed/sirv/tree/master/packages/sirv-cli#network-access:~:text=Using%20%2D%2Dhost%20without%20a%20value%20is%20equivalent%20to%20%2D%2Dhost%200.0.0.0)